doc/source/whatsnew/v0.25.0.rst

```diff
@@ -141,6 +141,7 @@ Bug Fixes
 - Bug in :func:`to_datetime` which would raise an (incorrect) ``ValueError`` when called with a date far into the future and the ``format`` argument specified instead of raising ``OutOfBoundsDatetime`` (:issue:`23830`)
 - Bug in an error message in :meth:`DataFrame.plot`. Improved the error message if non-numerics are passed to :meth:`DataFrame.plot` (:issue:`25481`)
 - Bug in error messages in :meth:`DataFrame.corr` and :meth:`Series.corr`. Added the possibility of using a callable. (:issue:`25729`)
+- Bug in `TextReader()` not properly working with UTF8 on Windows on Python 3.6+ (fix for :issue:`15086` instead of a workaround)
```

Comment:
move to I/O and replace this with 

Bug in ``:func:`read_csv` `` not properly interpreting the UTF8 encoded filenames on Windows on Python 3.6+ (:issue:`15086` )

Reply:
Fixed